frontend/src/static/js/v_summary.js

```diff
@@ -492,13 +492,52 @@ window.vSummary = {
       user.dailyCommits.forEach((commit) => {
         const { date } = commit;
         if (date >= sinceDate && date <= untilDate) {
-          user.commits.push(commit);
+          const filteredCommit = JSON.parse(JSON.stringify(commit));
+          if (this.filterBreakdown) {
```

Comment:
Should we filter by checked file types even when `filterBreakdown` is not checked? 

i.e. Should the user be able to check `filterBreakdown`, uncheck some file types, and then uncheck `filterBreakdown`, and still view the filtered ramp chart?

Reply:
No I don't think we should filter by checked file types when `filterBreakdown` is not checked.